webrender/src/prim_store.rs

```diff
@@ -2473,6 +2473,7 @@ impl PrimitiveStore {
                 if new_local_rect != metadata.local_rect {
                     metadata.local_rect = new_local_rect;
                     frame_state.gpu_cache.invalidate(&mut metadata.gpu_location);
+                    pic_state.local_rect_changed = true;
```

Comment:
any reason we can't just invalidate `pic_state.extra_gpu_data_handle` right here?

Reply:
```extra_gpu_data_handle``` exists on the ```Picture``` struct, not in ```PictureState```. We *do* have access to the picture here, and *could* invalidate it here - however, the way that ```extra_gpu_data_handle``` is used depends on the type of ```Picture```, and so it seems tidier to have that done in the code that deals with specific picture types, rather than adding a new ```match``` here on the picture type.